### DISCUSSION TITLE
Better image preview display

### DISCUSSION THREAD

**Discussion by zippomode** [directus/directus#17650] [Archive]
### Summary

<img width="836" alt="image" src="https://user-images.githubusercontent.com/9381877/221910585-8046dcca-a216-4ef2-8083-bbf0078fb7de.png">
I think the image preview should match how we see it in the editor.
With the same ratio, without cropping.
The current implementation forces the full image to be opened. Only then can you see if the image that the end user will see is correct.

### Basic Example

<img width="506" alt="image" src="https://user-images.githubusercontent.com/9381877/221913934-b031b5c1-cecc-4858-af3b-85c02fcff9ca.png">
How editor looks
<img width="284" alt="image" src="https://user-images.githubusercontent.com/9381877/221914058-f754cda2-df40-4954-a54e-ccbf42cb75bc.png">
How final image looks

No additional clicks to see how final image will look.
Current directus hides it from content creator.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/9381877/221918415-b41413cc-6f38-4a59-b171-69155cba371c.png">
Strapi (dislike it) shows preview as original file.

### Motivation

The current implementation forces the full image to be opened. Only then can you see if the image that the end user will see is correct.
As a result, the number of possible errors with images increases: it is impossible to unequivocally tell whether the picture is correct from the preview, and checking it every time is a waste of time.

### Detailed Design

The way the image looks in the editor is how the preview should look.
If 5:4 is specified in the editor, then the preview should also have this ratio. No cropping or other distortion of how the end user will see the image.
If the image is vertical, it should stay that way. Now the central part is being trimmed.
If the original image is not changed in the editor - show a preview of the original image.

### Must Have:
- Zero clicks to understand how final user will see an image.

### Drawbacks

It should be easy i think.

### Alternatives

The best option - if content creator can quickly customize how the previews look. Maybe inside of image field.

### Adoption Strategy

It will change only image display. And i think it will better. Current preview stay same with any image.

**Comment by rijkvanzanten** [comment]
This to me sounds like the "Crop to Fit" option in the Image interface. Have you tried that? 

![Screenshot 2023-02-28 at 9 55 43 PM](https://user-images.githubusercontent.com/9141017/222033621-f9e98936-d671-4f2f-be29-3f1e47ad01d5.png)

**Reply by zippomode** [reply]
Nice! Its better. Thanx.
